### PR TITLE
[14][MIG] sale order price recalculation

### PR DESCRIPTION
**Pull request by xavier-bouquiaux**
I saw a debate on 2 PR for this module 
https://github.com/OCA/sale-workflow/pull/1245 & https://github.com/OCA/sale-workflow/pull/1239
I based this one on 1245

@rousseldenis  FYI

### CODE REVIEW

**Comment by francesco-ooops** [comment]
@rousseldenis @xavier-bouquiaux any plan to fix/merge this one?

File: sale_order_price_recalculation/views/sale_order_view.xml [hunk 1]
```diff
@@ -5,7 +5,11 @@
         <field name="model">sale.order</field>
         <field name="inherit_id" ref="sale.view_order_form" />
         <field name="arch" type="xml">
-            <field name="note" position="after">
+            <xpath
+                expr="//page[@name='order_lines']/group[@name='note_group']/group/field[@name='note']"
+                position="after"
+            >
+            <!--<field name="note" position="after">-->
```
Comment (rousseldenis) [inline sale_order_price_recalculation/views/sale_order_view.xml:12]: @xavier-bouquiaux Could you please remove this ?

**Comment by pedrobaeza** [comment]
Isn't this now something that comes in the standard?

**Comment by nilshamerlinck** [comment]
Hello @pedrobaeza
- since [odoo 14.0](https://github.com/odoo/odoo/pull/31535), a button will show up indeed if pricelist has been changed:
image
- but this module is still useful imho to _Recalculate prices_ or _Reset descriptions_ regardless of pricelists :-)

Now the pending question is which approach should prevail:
- as in 12.0 and as currently merged in 13.0, without `onchange_helper`?
- or as in this PR: using `onchange_helper`?

@rousseldenis @pedrobaeza do you guys think you could come to a consensus on this topic? The goal is just to be in a position to work on the migration of this module for v15 :-) Thank you.

**Comment by pedrobaeza** [comment]
I think this should use the same approach as Odoo core, isn't it?

**Comment by xavier-bouquiaux** [comment]
I see that the modules exists in branch 14
i close this PR

**Pull request closed without merging**